### ISSUE TITLE
YAML should define if metric is required to be considered a compliant implementation

### ISSUE THREAD

**Issue by jamesmoessis** [open-telemetry/build-tools#132]
This stems from https://github.com/open-telemetry/opentelemetry-specification/issues/2972 and https://github.com/open-telemetry/opentelemetry-specification/pull/3158/files 

> "The [HTTP metrics semantic convention spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md) should provide clarity "what is considered a compliant implementation", e.g. "an implementation is considered compliant if it has implemented all of the metrics described by this spec"

**Comment by Oberon00** [comment]
I don't think this applies to traces. For traces, we have required attributes, but either you track a certain operation or not. We can't say "if you are not creating HTTP spans, then you are not compliant". Either you have an HTTP instrumentation enabled. Then it hopefully creates spans following HTTP conventions. Or you don't, then that's fine as well.
